src/common/globals.js

```diff
@@ -265,24 +269,40 @@ define([
             valueType: 'section'
         });
 
-        const storageMetadata = Storage.getAvailableBackends().map(id => Storage.getStorageMetadata(id));
+        if(pluginName === IMPORT_PLUGIN){
```

Comment:
It might be nicer to pass the storage backends into the function rather than filtering iff the import plugin is used.

Reply:
This function has been renamed to `pushStorageConfigMetadata`. Which takes in metadata for the plugin as well as `backends` to include in the config.